bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ng-at-internet",
-  "version": "2.2.1",
+  "version": "2.3.0",
```

Comment:
do not increment version in bower.json (and package.json). This will be done automatically when we will launch the release